.clusterfuzzlite/build.sh

```
@@ -2,17 +2,33 @@
+pip3 install .
 
-# Copy fuzz targets to the output directory
-# Each Python file in fuzz/ directory becomes a fuzz target
-for fuzzer in $SRC/rag_processor/fuzz/fuzz_*.py; do
-    if [ -f "$fuzzer" ]; then
-        fuzzer_basename=$(basename -s .py $fuzzer)
-        cp $fuzzer $OUT/$fuzzer_basename
-        chmod +x $OUT/$fuzzer_basename
-    fi
+# Build each fuzzer
+for fuzzer in $(find $SRC/rag_processor/fuzz -name 'fuzz_*.py'); do
```

Shellcheck (0.11.0)

[warning] For loops over find output are fragile. Use find -exec or a while read loop.

(SC2044)